subprojects/docs/src/docs/userguide/five_things.adoc

```diff
@@ -0,0 +1,112 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[five_things]]
+= Five things you need to know about Gradle
+
+Gradle is a flexible and powerful build tool that can easily feel intimidating when you start to learn it. There is potentially a lot to learn. However, there are some core principles that will make it easier to learn how to use Gradle effectively once you understand them. We present those principles here.
+
+== 1. Gradle is a general-purpose build tool
+
+Gradle allows you to build any software, because it makes few assumptions about what you're trying to build or how it should be done. The most notable restriction is that dependency management currently only supports Maven- and Ivy-compatible repositories and the filesystem.
+
+This doesn't mean you have to do a lot of work to create a build. Gradle makes it easy to build common types of project — say Java libraries — by adding a layer of conventions and prebuilt functionality through <<plugins#plugins,_plugins_>>. You can even create and publish custom plugins to encapsulate your own conventions and build functionality.
```

Comment:
:x: "types of project_s_"

Reply:
Would you say "projects" would be considered incorrect in British English? I do think this is considered fairly incorrect for American-style English.

Reply:
Yes, it sounds very odd. It seems to be a cultural thing, as witnessed by some of the comments on the [English StackExchange](https://english.stackexchange.com/questions/5539/types-of-things-vs-types-of-thing). I'll try to remember this for future reference, even if it hurts my internal ear :smile: